chapters/package-information.md

```diff
@@ -1445,7 +1445,7 @@ The metadata for the Primary Package Purpose field is shown in Table 36.
 | Attribute | Value |
 | --------- | ----- |
 | Required | No |
-| Cardinality | 0..* |
+| Cardinality | 0..1 <br> NOTE: The cardinality was incorrectly printed as 0..\* in the original publishing of the 2.3 specification |
```

Comment:
One very minor formatting suggestion is to [italicize](https://www.markdownguide.org/basic-syntax/#italic) the note (e.g. add an underscore before `Note` and after `specification`).

Reply:
Sounds good. I'll update that.